samples/utils.js

```diff
@@ -2,7 +2,7 @@
 
 (function(global) {
 	var Samples = global.Samples || (global.Samples = {});
-	var Color = global.Color;
+	var Color = global.Chart ? global.Chart.helpers.color : function() {};
```

Comment:
@simonbrunel does this one need to work when there is no `global.Chart`?

Reply:
the `utils.js` is also imported in the generic `index.html` page which hasn't chart.js included. Thus the check